lxd/storage/drivers/driver_zfs.go

```diff
@@ -271,7 +271,7 @@ func (d *zfs) Create() error {
 		}
 
 		// Create the zpool.
-		_, err = shared.RunCommand("zpool", "create", "-m", "none", "-O", "compression=on", d.config["zfs.pool_name"], loopPath)
+		_, err = shared.RunCommand("zpool", "create", "-m", "none", "-O", "compression=on", "-O", "feature@async_destroy=disabled", d.config["zfs.pool_name"], loopPath)
```

Comment:
`async_destroy` is apparently supported even by [ZFS 0.7](https://openzfs.github.io/openzfs-docs/man/v0.7/5/zpool-features.5.html) so that's probably safe to do unconditionally.

Reply:
Cool that sounds like a plan

Reply:
Does that mean we dont need the -f flag ?

Reply:
It's not yet clear what the actual issue is, it might just be this async feature, or it might be something else.

I have this [draft PR](https://github.com/canonical/microcloud/pull/310) up now which should help us elucidate the state of the VMs prior to the test running at least.

Reply:
Thanks, will hold off on merging this until you gey your results.